### PR TITLE
Initial generated Azure.Data.Tables client

### PR DESCRIPTION
**Pull request by christothes**
The focus of this PR is to standup some initial rough scaffolding for Azure.Data.Tables. This also intends to validate CI builds run as expected.

### CODE REVIEW

File: sdk/tables/Azure.Data.Tables/Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (pakrym) [inline sdk/tables/Azure.Data.Tables/Directory.Build.props:1]: Remove this file, it doesn't define anything except inclusion of higher level `Directory.Build.props` that happens automatically.

File: sdk/tables/Azure.Data.Tables/src/Azure.Data.Tables.csproj [hunk 2]
```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This client library enables working with the Microsoft Azure Storage Table service</Description>
+    <AssemblyTitle>Microsoft Azure.Storage.Tables client library</AssemblyTitle>
+    <Version>12.5.0-preview.1</Version>
```
Comment (pakrym) [inline sdk/tables/Azure.Data.Tables/src/Azure.Data.Tables.csproj:5]: Not sure about this version

File: sdk/tables/Azure.Data.Tables/src/Azure.Data.Tables.csproj [hunk 3]
```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This client library enables working with the Microsoft Azure Storage Table service</Description>
+    <AssemblyTitle>Microsoft Azure.Storage.Tables client library</AssemblyTitle>
```
Comment (pakrym) [inline sdk/tables/Azure.Data.Tables/src/Azure.Data.Tables.csproj:4]: Storage shouldn't be in the title

File: sdk/tables/Azure.Data.Tables/src/StorageSharedKeyCredential.cs [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+using System.Text;
+using System.Threading;
+
+namespace Azure.Data.Tables
+{
+    /// <summary>
+    /// A <see cref="StorageSharedKeyCredential"/> is a credential backed by
+    /// a Storage Account's name and one of its access keys.
+    /// </summary>
+    public class StorageSharedKeyCredential
```
Comment (pakrym) [inline sdk/tables/Azure.Data.Tables/src/StorageSharedKeyCredential.cs:15]: TablesSharedKeyCredential

**Comment by pakrym** [comment]
I don't remember if you have to merge your pipeline definitions first for automation to pick it up.

**Comment by christothes** [comment]
> I don't remember if you have to merge your pipeline definitions first for automation to pick it up.

I think that is correct - Here are the stand alone yml files. #11502

**Comment by christothes** [comment]
closing and re-opening a new PR to pick up the new pipeline

**Pull request closed without merging**